swift-mafia/Mafia/Mafia/StopWatch/StopWatch.swift

```diff
@@ -0,0 +1,65 @@
+import SwiftUI
+
+struct StopWatchContentView: View {
+    @ObservedObject var stopwatch = Stopwatch()
```

Comment:
@State

Reply:
We want the stopwatch to reinit when it appears again so using @State is not required.